_includes/header.html

```diff
@@ -55,6 +55,8 @@
              Subprojects
           </a>
           <div class="dropdown-menu" aria-labelledby="navbarDropdownSubprojects">
+            <a class="dropdown-item" href="{{ site.baseurl }}/docs/format/Flight.html">Arrow Flight</a>
+            <a class="dropdown-item" href="{{ site.baseurl }}/docs/dev/format/FlightSql.html">Arrow Flight SQL</a>
```

Comment:
We need to remove `dev/` here after we release 8.0.0, right?

Reply:
Yes, good point. I filed ARROW-15937 as a reminder.